rosco-web/config/rosco.yml

```diff
@@ -202,25 +202,25 @@ openstack:
   bakeryDefaults:
     identityEndpoint: ${OS_AUTH_URL:}
     domainName: Default
-    floatingIpPool: ext_vlan1472_net
+    floatingIpPool: ext_network
```

Comment:
I didn't know that these are upstream driven configuration ... Fine with leaving it as a generic default.